Realm/RLMObjectBase.mm

```diff
@@ -88,9 +81,9 @@ - (instancetype)initWithRealm:(__unsafe_unretained RLMRealm *const)realm
     if (self) {
         _realm = realm;
         _objectSchema = schema;
-        if (useDefaults) {
+        if (useDefaults && !schema.isSwiftClass) {
```

Comment:
afaict `useDefaults` is only used in the standalone init case, so it seems simpler to just move all this to that method.